frontend/src/libs/components/profile-settings-sidebar/profile-settings-button/styles.module.scss

```diff
@@ -0,0 +1,45 @@
+.profile-button {
+  display: flex;
+  align-items: center;
+  min-width: 317px;
+  margin-bottom: 8px;
+  padding: 3px 26px 3px 3px;
+  background-color: var(--white-alpha-80);
+  border: none;
+  border-radius: 20px;
+  box-shadow: 0 4px 23px var(--violet-100);
+  cursor: pointer;
+}
+
+.focused {
+  box-shadow: 0 4px 49px 0 var(--blue-alpha-90);
+}
+
+.profile-button-text {
+  display: block;
+  width: 145px;
```

Comment:
it's better to avoid using fixed width and height values if it's possible

Reply:
I might need to keep it, I needed the text not to break trough the specific line, I'll try to see what I can do

Reply:
could you add screenshots on what you are trying to achieve, what If text exceeds this line? looks like in this case it should be moved to the next line and we don't need width for this